command: /usr/bin/python /u/pedronis/PyPy/dist/pypy/tool/alarm.py 900 /u/pedronis/PyPy/dist/pypy/interpreter/py.py --oldstyle /u/pedronis/PyPy/dist/lib-python-2.3.4/test/test_ioctl.py
run by: pedronis@ratthing-b246
sys.platform: linux2
sys.version_info: (2, 3, 5, 'final', 0)
startdate: Thu Apr 28 01:34:17 2005
pypy-revision: 11549
============================================================
faking <type 'unicode'>
faking <type 'module'>
faking <type 'file'>
faking <type 'posix.stat_result'>
faking <type 'posix.statvfs_result'>
Traceback (application-level):
  File "/u/pedronis/PyPy/dist/lib-python-2.3.4/test/test_ioctl.py", line 2 in ?
from test_support import TestSkipped, run_unittest
  File "test_support.py", line 4 in ?
    raise ImportError, 'test_support must be imported from the test package'
ImportError: test_support must be imported from the test package
========================== closed ==========================
execution time: 8.79495596886 seconds
exit status: 1
